viz/visualize.py

```diff
@@ -43,10 +43,10 @@ def reconstructions(self, data, size=(8, 8), filename='recon.png'):
         self.model.train()
 
         # Upper half of plot will contain data, bottom half will contain
-        # reconstructions
-        num_images = size[0] * size[1] / 2
-        originals = input_data[:num_images].cpu()
-        reconstructions = recon_data.view(-1, *self.model.img_size)[:num_images].cpu()
+        # reconstructions of the original image
+        num_images = size[0]
+        originals = input_data.cpu()
```

Comment:
Do these always get sent to cpu? Does it not conflict with `input_data = data.to(self.model.device)` where the `model.device` determines where it is sent? I'm familiar enough with PyTorch to know this.

Reply:
Seems so. I left the ```.cpu()``` call operating in the way that I found it.

Reply:
@linesd @alecokas when you train or test the model you want all to be on your most powerful device to make it quicker, but when you save it has to be on CPU such that anyone can read it! Good  question though :)